<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Current Sgtk Environment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="path_layout">
     <item>
      <widget class="QLabel" name="path_label">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path_lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="path_btn">
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="add_delete_layout">
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset_btn">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>453</width>
        <height>104</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>451</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="replace_box_layout"/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="check_option_layout">
       <item>
        <widget class="QCheckBox" name="folder_name_change_option_checkbox">
         <property name="text">
          <string>All folder inside folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="all_files_inside_folder_checkbox">
         <property name="text">
          <string>All files inside folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="btn_layout">
       <item>
        <widget class="QPushButton" name="space_change_to_underbar_btn">
         <property name="text">
          <string>Space -&gt; UnderBar</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="replace_cancel_btn_layout">
         <item>
          <widget class="QPushButton" name="replace_btn">
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_btn">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>path_lineEdit</tabstop>
  <tabstop>path_btn</tabstop>
  <tabstop>add_btn</tabstop>
  <tabstop>delete_btn</tabstop>
  <tabstop>reset_btn</tabstop>
  <tabstop>folder_name_change_option_checkbox</tabstop>
  <tabstop>all_files_inside_folder_checkbox</tabstop>
  <tabstop>space_change_to_underbar_btn</tabstop>
  <tabstop>replace_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
